OutOfSchool/OutOfSchool.Common/Validators/CustomAgeAttribute.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace OutOfSchool.Common.Validators;
+
+/// <summary>
+/// Validation attribute for dates. It allows only dates that are at least <see cref="MinAge"/> and at most <see cref="MaxAge"/>.
+/// This attribute can be used on <see cref="DateTime"/> or <see cref="DateOnly"/> types.
+/// </summary>
+[AttributeUsage(AttributeTargets.Property | AttributeTargets.Field | AttributeTargets.Parameter, AllowMultiple = false)]
+public class CustomAgeAttribute : DataTypeAttribute
+{
+    /// <summary>
+    /// The approximate number of days in a year including leap years.
+    /// </summary>
+    public const double ApproximateDaysInYear = 365.2425;
```

Comment:
(Redundant code) Is `ApproximateDaysInYear` still used?

Reply:
> (Redundant code) Is `ApproximateDaysInYear` still used?

Forgot to remove, fixed